src/layout/Input/InputComponent.tsx

```diff
@@ -42,7 +45,7 @@ export function InputComponent({ node, isValid, formData, handleDataChange }: II
           required={required}
           value={value}
           aria-describedby={textResourceBindings?.description ? `description-${id}` : undefined}
-          formatting={formatting as IInputFormatting}
+          formatting={reactNumberFormatConfig as IInputFormatting}
```

Comment:
Can we type the "useMapToReactNumberConfig"-hook to have the return type of IInputFormatting?

Reply:
Done!